exercises/practice/paasio/test_utils.py

```diff
@@ -0,0 +1,150 @@
+import errno
+import io
+import inspect
+import logging
+import os
+
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
`LOGGER` not referenced

```suggestion
import os


```